src/qt/rpcconsole.cpp

```diff
@@ -906,12 +906,8 @@ void RPCConsole::on_lineEdit_returnPressed()
 
         cmdBeforeBrowsing = QString();
 
-        WalletModel* wallet_model{nullptr};
 #ifdef ENABLE_WALLET
-        const int wallet_index = ui->WalletSelector->currentIndex();
-        if (wallet_index > 0) {
-            wallet_model = ui->WalletSelector->itemData(wallet_index).value<WalletModel*>();
-        }
+        WalletModel* wallet_model = ui->WalletSelector->currentData().value<WalletModel*>();
```

Comment:
Why it safe to remove `if (wallet_index > 0) {`?

Reply:
It's safe because what we want is the current `Qt::UserRole` data as `WalletModel*`. If there is none then `currentData().value<WalletModel*>()` returns `nullptr`, which is the case for the fist item.

Reply:
Why was `{}` syntax for `wallet_model` initialization dropped? It could prevent an implicit data conversion.

Reply:
You mean this should be `WalletModel* wallet_model{ui->WalletSelector->currentData().value<WalletModel*>()};`?

Reply:
Yes. It seems more robust.